R/derive_extreme_records.R

```diff
@@ -95,6 +95,13 @@
 #'
 #'   *Permitted Values*: a condition
 #'
+#' @param keep_vars_source Variables to be kept in the new records
+#'
+#'   A named list returned by `exprs()` defining the variables to be kept for the
+#'   new records.
+#'
+#'   *Default*: `everything()`
```

Comment:
```suggestion
```

We are no longer mentioning defaults in the roxygen, but also in this instance you actually set the default to `NULL` in the function header and then within the code it is set to `exprs(everything())` if user didn't explicitly specify. 

I think it would be important to mention to the user that the default is to everything but then in the examples if we're using the default just don't show the argument in the function call? What're your thoughts @bundfussr